conf/messages.de

```diff
@@ -32,6 +32,7 @@ securesocial.signup.thankYouCheckEmail=Vielen Dank. Bitte prüfen Sie ihr E-Mail
 securesocial.signup.invalidLink=Der Link ist ungültig
 securesocial.signup.signUpDone=Vielen Dank für Ihre Registrierung.
 securesocial.signup.invalidPassword=Das Passwort muss mindestens {0} Zeichen, eine Ziffer, einen Großbuchstaben und einen Kleinbuchstaben enthalten.
+securesocial.signup.passwordInfoOnChange=Das neue Passwort muss mindestens {0} Zeichen, eine Ziffer, einen Großbuchstaben und einen Kleinbuchstaben enthalten.
```

Comment:
It is needed yes. Because the user is asked to at the new password and not a password. Sorry I have overread it.